src/bot.py

```diff
@@ -85,6 +73,28 @@ def process_recent_comments(reddit, server):
         # Sleep for five seconds.
         await asyncio.sleep(5)
 
+async def comment_loop(reddit, server):
+    """The bot's main Reddit comment loop."""
+    # Let's make a tick twice every day.
+    tick_duration = 12 * 60 * 60
+    
+    # TODO: handle multiple subreddits.
+    for comment in reddit.subreddit('simeconomy').stream.comments(pause_after=0):
+        # Prcoess next comment if necessary.
+        if not (comment is None or is_comment_replied_to(reddit, comment)):
+            if comment.body.startswith(prefix):
+                process_comment(comment, server)
+                await asyncio.sleep(10 * 60)
+        
+        # Notify the server that one or more ticks have elapsed if necessary.
+        time_diff = int(time.time() - server.last_tick_timestamp)
+        if time_diff > tick_duration:
+            for i in range(time_diff // tick_duration):
+                server.notify_tick_elapsed()
```

Comment:
I don't think this logic needs to be duplicated. Notifying the server that a tick has elapsed executes pending recurring transfers. I don't think that this duplicated code will cause ticks to occur twice due to the way it was written, but it is kind of confusing to duplicate it. Maybe separate it out into its own loop? You could even sleep for `tick_duration` seconds in that loop if you created it.